ZupIT/ritchie-cli
# Added new version warning for repositories in rit helper

**- What I did**

Added new version notice available for a repository.

> ![rit](https://user-images.githubusercontent.com/67295691/95199470-3da9a300-07b3-11eb-9239-cd04eead761b.png)


**- How to verify it**

> Clone this pr with the command
```bash
git fetch upstream pull/582/head:improvement/added-new-version-notice-in-helper
git checkout improvement/added-new-version-notice-in-helper
make build && cp dist/linux/rit /usr/local/bin/
```
> Add a test repository with an outdated version (eg https://github.com/brunats/brunats-formulas and version v1.0.3)
```bash
rit add repo
rit list repo # Check the repo added
```

> Check the new version notice in the repository name prefix
```bash
rit
```

**- Description for the changelog**


Added new version warning for repositories in rit helper
